mecode/main.py

```diff
@@ -81,7 +81,7 @@ def decode2To3(s):
 class G(object):
 
     def __init__(self, outfile=None, print_lines=True, header=None, footer=None,
-                 aerotech_include=True, output_digits=6, direct_write=False,
+                 aerotech_include=False, output_digits=6, direct_write=False,
```

Comment:
This library was originally built for members of my lab, which exclusively uses Aerotech machines.

I regret defaulting to it, but to switch now would break many of their prints.

I've been thinking for a while of releasing a 1.0 version of mecode that can make breaking changes, and at that time I would reach out to the current lab members and help them through the transition.

That is a long way of saying I can't merge this default change now, but am willing to merge this and some other breaking changes in a near term future release.